src/site/_includes/components/LanguageList.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const {getTranslatedUrls} = require('../../_filters/urls');
+const {i18n} = require('../../_filters/i18n');
+const {languageNames} = require('../../../../shared/locale');
+
+module.exports = (url, lang) => {
+  const langhrefs = getTranslatedUrls(url)
+    .filter((langhref) => langhref[0] !== lang)
```

Comment:
I'm not sure what's going on here but e.g. on a [non-English page](https://deploy-preview-6259--web-dev-staging.netlify.app/i18n/zh/vitals/), we don't show English in the list.

This is confusing because it _is_ being rendered in the meta tags.

Reply:
We don't have a mechanism to force an English version of the page atm. Link web.dev.i18n/en/foo does not exist, and web.dev/foo will serve whichever language the browser is set to, defaulting to English. In meta tags it does not matter, because robots usually do not have language preference. 

This is I was reluctant to link to English, as there is no way to reliably point to "en" version of the page. This said, I think the widger do add value by making the other translations more discoverable, so I opted to still send a PR with this solution.

Reply:
`web.dev/i18n/en/foo` does exist, but only in prodution. There's a gulp (I think) task that copies over all of the content not in the `i18n` folder to an en folder. This was because of when you have english as the primary language of the browser and spanish as the secondary firebase would try to serve spanish first.

Reply:
Ah sweet! I totally forgot about this quirk. Thanks @MichaelSolati !